src/frontend/components/administration/apikey/ApikeyTab.vue

```diff
@@ -5,14 +5,37 @@
       <v-spacer />
       <v-btn
         icon
-        @click.stop="setVisible(true)"
+        @click.stop="setVisible('importDialogVisible', true)"
```

Comment:
You could directly mutate data and drop `setVisible` (as it's not only for visible values).
```suggestion
        @click.stop="this.importDialogVisible = true"
```

Same for all dialogs